shell/packages/sandstorm-ui-powerbox/powerbox-server.js

```diff
@@ -112,6 +112,69 @@ Meteor.methods({
     const sturdyRef = waitPromise(saveFrontendRef(frontendRefVariety, apiTokenOwner, requirements)).sturdyRef.toString();
     return { sturdyRef, descriptor };
   },
+
+  fulfillUiViewRequest(identityId, grainId, petname, roleAssignment, saveLabel, ownerGrainId) {
+    check(identityId, String);
```

Comment:
Either comment which inputs are type-checked elsewhere or check them here too to be safe.